tests/fixtures/issues/issue-1219.php

```diff
@@ -8546,6 +8546,7 @@
             'Version' => '0.0',
             'MajorVer' => '0',
             'MinorVer' => '0',
+            'Platform' => 'Android',
```

Comment:
My comparison project caught this, apparently the `Platform` property was missing from this test.